### PR TITLE
Partial fix for the 0.1-p1 release

### PR DESCRIPTION
**Pull request by dhylands**
This addresses an issue where loadAdapters gets called twice, once when creating the http server and a second time when starting the https server.

### CODE REVIEW

File: adapter-manager.js [hunk 1]
```diff
@@ -357,11 +364,16 @@ class AdapterManager extends EventEmitter {
...
+    if (!this.adaptersLoaded) {
+      // The adapters are not currently loaded, no need to unload.
+      return;
+    }
     for (var adapterId in this.adapters) {
       var adapter = this.adapters[adapterId];
       console.log('Unloading', adapter.name);
       adapter.unload();
     }
+    ths.adaptersLoaded = false;
```
Comment (hobinjk) [inline adapter-manager.js]: Typo, s/ths/this

**Comment by hobinjk** [comment]
Otherwise LGTM. I think you can point this PR at the 0.1-p1 branch using GitHub, though I don't remember the exact location of that option

**Comment by dhylands** [comment]
Yeah - I think I need to close this PR and create another one in order to get it branched properly.

**Comment by dhylands** [comment]
This PR was superceded by #274

**Pull request closed without merging**